### ISSUE TITLE
base_config

### ISSUE THREAD

**Issue by mikasa-2024** [swc-17/SparseDrive#13]
Hi, there, I want to do some experiments with backbone res-101, I have changed the matched resolution, data_aug and learning rate, but the training process is not stable, gradient explosion occurs sometimes, can u give me some tips about scale up your model? 
Tanks a lot!
(If connect with e-mail is comfortable for u, my address is hooz1009@163.com )
![Screenshot 2024-07-23 at 2 01 07 PM](https://github.com/user-attachments/assets/bf5db77d-0e61-47d4-bd00-02cb3184db6e)

**Comment by swc-17** [comment]
Could you provide more detaild information about your modification? You can refer [this issue](https://github.com/swc-17/SparseDrive/issues/14) first.

**Comment by mikasa-2024** [comment]
I have modified the corresponding input size and data_aug_conf, and used the backbone_config like sparse4dv3, it seems like my model can train stably. (Maybe the reason before is not loading checkpoints correctly and setting requires_grad=TRUE)
![Screenshot 2024-07-25 at 10 41 55 AM](https://github.com/user-attachments/assets/5d2d72bb-3a43-4eed-a29c-53d6536ea0b7)

Thanks!

**Comment by qwert1887** [comment]
@mikasa-2024 Hi,Bro! Should you share how you modified your data aug and learning rate? By the way, have you trained the stage 2 successfully?
